lib/server.js

```diff
@@ -404,19 +403,18 @@ Server.prototype.onWebSocket = function(req, socket){
  * @api public
  */
 
-Server.prototype.attach = function(server, options){
+Server.prototype.attach = function (server, options) {
   var self = this;
-  var options = options || {};
+  options = options || {};
   var path = (options.path || '/engine.io').replace(/\/$/, '');
 
-  var destroyUpgrade = (options.destroyUpgrade !== undefined) ? options.destroyUpgrade : true;
   var destroyUpgradeTimeout = options.destroyUpgradeTimeout || 1000;
```

Comment:
**This is the diff comment that requires attention**

This variable is not actually used, but it seems as if it *should* be used, looking at line 450.